docs/babel.md

```diff
@@ -0,0 +1,36 @@
+## Usage with Babel
+
+The Babel Plugin is highly recommended, but not required in version 8 and above.
+
+See the [installation instructions](install.md).
+
+### Features which are enabled with the babel plugin
+
+### styled.element Syntax
+`styled('div')` will work without the plugin
```

Comment:
I'd like to see an explanation of the two syntax types and the reason for using one over the other. It probably should go in the 'getting started' section (if there is one..) though

Reply:
I have a getting started doc on deck for my next PR. I'll need some help understanding the big differences between syntaxes